src/styles/eleventy-patterns/patterns/nav.scss

```diff
@@ -1,9 +1,145 @@
+.EP_header-container {
+  background: var(--EP_color-white);
+  box-shadow:
+    0 0 1px rgba(67, 90, 111, 0.47),
+    0 2px 4px -2px rgba(67, 90, 111, 0.3);
+  position: fixed;
+  top: 0;
+  transition: transform var(--transition-quick);
+  width: 100%;
+  z-index: 1;
+
+  @media (min-width: 800px) {
```

Comment:
We should use `em`s for breakpoints. And maybe store them somewhere

Reply:
I updated all media queries to use `em`s, but it isn't stored anywhere.

Reply:
Perfect, thanks. Would you mind opening a Github issue to standardize breakpoints in custom properties or SCSS vars?